python/common/org/python/types/Dict.java

```diff
@@ -357,10 +357,12 @@ public void __delitem__(org.python.Object item) {
 
     @org.python.Method(
             __doc__ = "",
-            args = {"iterable"},
-            default_args = {"value"}
+            default_args = {"iterable", "value"}
```

Comment:
I see that the `iterable` arg has been made a default arg, with accompanying `iterable == null` check – is this exercised by any test? It looks like it will raise a TypeError exception, but I _think_ the only TypeError being caught in the tests is for a non-iterable, not for a missing iterable.